articles/index.yml

```diff
@@ -22,107 +22,116 @@ landingContent:
-            url: quickstart.md            
-          - text: Learn about impactful quantum programs with Q#
-            url: quickstarts/search.md
-          - text: Learn about solving our biggest challenges in chemistry
-            url: https://cloudblogs.microsoft.com/quantum/2018/12/04/simulating-nature-with-the-new-microsoft-quantum-development-kit-chemistry-library/
       - linkListType: learn
         links:
-          - text: Learn with the Quantum Katas
-            url: intro-to-katas.md
-          - text: Browse Q# samples
-            url: https://docs.microsoft.com/samples/browse/?languages=qsharp
+          - text: What is quantum computing?
+            url: overview/what-is-quantum-computing.md
+          - text: What can a quantum computer do?
+            url: overview/quantum-computers.md
+          - text: Why learn quantum computing?
+            url: overview/why-learn-quantum-computing.md
+          - text: What is Q#?
+            url: overview/what-is-qsharp.md
+          - text: How can I learn about quantum computing?
+            url: overview/how-to-learn-quantum-computing.md
 
 
   # Develop
-  - title: Develop with Q# and Python or Jupyter notebooks
+  - title: Get the Quantum Development Kit
     linkLists:
-    - linkListType: how-to-guide
-      links:
-      - text: Python install
-        url: install-guide/index.md#develop-with-python
-      - text: Jupyter install
-        url: install-guide/index.md#develop-with-jupyter-notebooks
-    - linkListType: how-to-guide
+    - linkListType: download
       links:
-      - text: Q# Samples with Python
-        url: https://docs.microsoft.com/samples/browse/?languages=python&products=qdk
-      - text: Teleport Q# Jupyter notebook
-        url: https://docs.microsoft.com/samples/microsoft/quantum/quantum-teleportation/
-      - text: How to build Q# Jupyter notebooks
-        url: https://docs.microsoft.com/samples/microsoft/quantum/intro-to-qsharp-jupyter/
+        - text: Install the QDK
+          url: install-guide/index.md
+        - text: Update the QDK
+          url: install-guide/update.md
 
 
   # Develop
-  - title: Q# development environments
+  - title: Quickstarts and tutorials
     linkLists:
-    - linkListType: how-to-guide
-      links:
-      - text: Visual Studio install
-        url: install-guide/index.md#develop-with-c-on-windows-using-visual-studio
-      - text: Visual Studio Code install
-        url: install-guide/index.md#develop-with-c-using-visual-studio-code
+      - linkListType: quickstart
+        links:
+          - text: Build a quantum random number generator
+            url: quickstarts/qrng.md
+          - text: Quantum basics with Q#
+            url: quickstart.md            
+          - text: "Quantum speedup: Grover's search algorithm"
+            url: quickstarts/search.md
+      - linkListType: tutorial
+        links:
+          - text: Qubit-level programming
+            url: tutorials/qft-tutorial.md
```

Comment:
We should leave this out until the tutorial is merged in. Right now this links to a page with nothing on it and I think it's probably more efficient to split these PRs.

Reply:
Makes sense. Just removed it, and made a note in the tutorial PR to add them in there